www/src/components/feedback-widget/buttons.js

```diff
@@ -0,0 +1,217 @@
+import { css } from "@emotion/core"
+import styled from "@emotion/styled"
+import { keyframes } from "@emotion/core"
+import {
+  mediaQueries,
+  colors,
+  fontSizes,
+  radii,
+  shadows,
+  space,
+} from "../../utils/presets"
+
+const rotation = keyframes`
+  0% {
+    transform: translateX(${space[1]}) rotate(0deg);
+  }
+  100% {
+    transform: translateX(${space[1]}) rotate(360deg);
+  }
+`
+
+export const focusStyle = css`
+  box-shadow: 0 0 0 0.12rem ${colors.accent};
```

Comment:
This focus style still isn't working in Safari. I played around with it a bit and opted to change it to use `outline` and `outline-offset` instead:
```
outline: 2px solid ${colors.accent};
outline-offset: -2px;
```
https://codepen.io/marcysutton/pen/WBxEom